### PR TITLE
First pass at image bundles

### PR DESCRIPTION
**Pull request by dannylamb**
**GitHub Issue**: Part of https://github.com/Islandora-CLAW/CLAW/issues/549

* Also part of https://github.com/Islandora-CLAW/CLAW/issues/490

* Blocks https://github.com/Islandora-CLAW/islandora_image/pull/11

# What does this Pull Request do?

Adds field storage for some fields that will be used across multiple bundles.

# What's new?
All config, all the time.

# How should this be tested?

You should test https://github.com/Islandora-CLAW/islandora_image/pull/11 after merging this.  This is its prerequisite.

# Interested parties
@Islandora-CLAW/committers

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -20,7 +20,8 @@
     "drupal/search_api": "^1.0@beta",
     "islandora/claw-jsonld": "dev-8.x-1.x",
     "stomp-php/stomp-php": "4.*",
-    "drupal/jwt": "1.0.0-alpha6"
+    "drupal/jwt": "1.0.0-alpha6",
+    "drupal/media_entity": "8.x-1.2"
```
Comment (jonathangreen) [inline composer.json]: Looks like composer is not happy with this version string.

**Comment by dannylamb** [comment]
Thanks for bumping this, @jonathangreen

**Comment by jonathangreen** [comment]
This is good to go once we get composer / travis all happy.

**Comment by dannylamb** [comment]
Sad times:
```
RuntimeException: Unable to determine class for field type 'image'
```
I'll need to play with the dependencies in the base kernel test.

**Comment by dannylamb** [comment]
I _think_ that's it.  I had to add all transitive dependencies manually to the base test setup.

**Comment by dannylamb** [comment]
We're green

**Pull request merged**